tools/prepare_release.csx

```diff
@@ -1,15 +1,18 @@
 #r ".\packages\Octokit.0.26.0\lib\net45\Octokit.dll"
 
 using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text.RegularExpressions;
 
 using Octokit;
 using Octokit.Helpers;
-using System.Runtime.CompilerServices;
 
 const string repoOwner = "FakeItEasy";
 const string repoName = "FakeItEasy";
 const string releaseFromBranchName = "develop";
 const string targetBranchName = "master";
+const string existingReleaseName = "vNext";
+const string nextReleaseName = existingReleaseName;

```

Comment:
Not strictly necessary, but do make it more convenient to test against different issues, not unlike being able to switch out repoOwner above.